static/js/app-dashboard/components/graphs/pieChart.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <Pie :data="chartData" :options="chartOptions" :plugins="[pieLabelsLine]"/>
+</template>
+
+<script lang="ts">
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js'
+import { Pie } from 'vue-chartjs'
+
+ChartJS.register(ArcElement, Tooltip, Legend)
+
+export default {
+  name: 'App',
+  components: {
+    Pie
+  },
+  props: {
+    labels: {
+      type: Array,
+      default: () => []
+    },
+    datasets: {
+      type: Array,
+      default: () => []
+    },
+    count: {
+      type: Array,
+      default: () => []
+    },
+    tooltipTitle: {
+      type: String,
+      default: () => ''
+    },
+    padding: {
+      type: Number,
+      default: () => 50
+    },
+  },
+  data() {
+    return {
+      pieLabelsLine: {
+        id: "pieLabelsLine",
+        afterDraw(chart) {
+          const {
+            ctx,
+            chartArea: { width, height },
+          } = chart;
+
+          const cx = chart._metasets[0].data[0].x;
+          const cy = chart._metasets[0].data[0].y;
+
+          const sum = chart.data.datasets[0].data.reduce((a, b) => a + b, 0);
+
+          chart.data.datasets.forEach((dataset, i) => {
```

Comment:
I'm curious about what `afterDraw` does here

Reply:
This outer labels part

<img width="191" alt="image" src="https://github.com/peermetrics/web/assets/34216326/e5aeff85-c85f-4eff-8ab2-158b04e5bb2d">